browser/src/shared/components/CodeViewToolbar.tsx

```diff
@@ -97,23 +98,21 @@ export const CodeViewToolbar: React.FunctionComponent<CodeViewToolbarProps> = pr
                 {
                     // Only show the "View file" button if we were able to fetch the file contents
                     // from the Sourcegraph instance
-                    !props.fileInfoOrError.baseCommitID &&
-                        (props.fileInfoOrError.content !== undefined ||
-                            props.fileInfoOrError.baseContent !== undefined) && (
-                            <li className={classNames('code-view-toolbar__item', props.listItemClass)}>
-                                <OpenOnSourcegraph
-                                    ariaLabel="View file on Sourcegraph"
-                                    className={props.actionItemClass}
-                                    iconClassName={props.actionItemIconClass}
-                                    openProps={{
-                                        sourcegraphURL: props.sourcegraphURL,
-                                        repoName: props.fileInfoOrError.repoName,
-                                        filePath: props.fileInfoOrError.filePath,
-                                        rev: props.fileInfoOrError.rev || props.fileInfoOrError.commitID,
-                                    }}
-                                />
-                            </li>
-                        )
+                    'blob' in props.fileInfoOrError && props.fileInfoOrError.blob.content && (
```

Comment:
This would now also evaluate to `false` for `content === ''` (empty files), whereas before it compared to `undefined`
```suggestion
                    'blob' in props.fileInfoOrError && props.fileInfoOrError.blob.content !== undefined && (
```